src/components/bottomSheetModal/BottomSheetModal.tsx

```diff
@@ -45,6 +45,7 @@ const BottomSheetModalComponent = forwardRef<
     snapPoints,
     enablePanDownToClose = true,
     animateOnMount = true,
+    bottomSheetWrapper: BottomSheetWrapper = React.Fragment,
```

Comment:
i would name it `containerComponent` , aligning with other provided components naming https://github.com/gorhom/react-native-bottom-sheet/blob/master/src/components/bottomSheet/types.d.ts#L258C6-L283